roles/tarzan/tasks/main.yml

```diff
@@ -16,9 +16,20 @@
               backup=no
   tags: tarzan
 
+- name: add the lua_path to funk_004's environment via the sourceme_upps script
+  lineinfile: dest={{ ngi_pipeline_conf }}/{{ bash_env_upps_script }}
+              line='export LUA_PATH={{ lua_path }}'
+              backup=no
```

Comment:
If I understand you correctly this writes the entire export line for the lua_path.
For charon & ngi_pipeline that export line always exists, the deployment merely adds the relevant paths.